share/completions/gpgbackup.fish

```diff
@@ -0,0 +1,356 @@
+#
```

Comment:
I think including this file might have been a mistake?

Reply:
Removed.